### ISSUE TITLE
PyROOT: AttributeError: Failed to get attribute TPyDispatcher from ROOT

### ISSUE THREAD

**Issue by op3** [root-project/root#6579]
- [x] Checked for duplicates

### Describe the bug
When I try to access `ROOT.TPyDispatcher` from PyROOT (for example, using tutorials/pyroot/gui_ex.py), the following error occurs:

```
input_line_37:1:10: fatal error: 'ROOT/TPyDispatcher.h' file not found
#include "ROOT/TPyDispatcher.h"
         ^~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/usr/share/doc/root/tutorials/pyroot/gui_ex.py", line 35, in <module>
    m = ROOT.TPyDispatcher( MyDraw )
  File "/usr/lib/python3.8/site-packages/ROOT/_facade.py", line 163, in _fallback_getattr
    raise AttributeError("Failed to get attribute {} from ROOT".format(name))
AttributeError: Failed to get attribute TPyDispatcher from ROOT
```

I also cannot find `TPyDispatcher.h` on my system, which PyROOT tries to [include here](https://github.com/root-project/root/blob/master/bindings/pyroot/pythonizations/python/ROOT/_facade.py#L292)

### Expected behavior

ROOT.TPyDispatcher is known to be broken in 6.22.00, but [it is supposed to be fixed for 6.22.02](https://github.com/root-project/root/pull/6065).

### To Reproduce

Run `python tutorials/pyroot/gui_ex.py`

### Setup

1. ROOT version: 6.22.02
2. Operating system: Linux x86_64 (Arch Linux and Ubuntu 18.04)
3. Python version: Python3.8
4. I tested this both for root from the Arch Linux repositories and from conda-forge, so I assume it is not the packagers fault.

**Comment by etejedor** [comment]
Thank you for the report, the installation of the header was missing, I made a PR to fix it.

**Comment by etejedor** [comment]
Fix also backport to 6.22 patches here: https://github.com/root-project/root/pull/6589
